new/LIT6948GenealogyMonks.xml

```diff
@@ -1,65 +0,0 @@
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
```

Comment:
let me remind you that you should not delete records created through the web app before these had been merged. It will otherwise stay in the web app but remain invisible in our shared github, but will still be searchable online.
OK for this time; I will delete it manually in the backend, but please try to stick to the workflow.
Please also check if any of the content should be used in LIT4983ListDabra